source/Glossary/ip_warmup.md

```diff
@@ -11,7 +11,7 @@ seo:
   keywords: IP, warm, up, warmup, ramp, ramping
 ---
 
-[IP warming]({{root_url}}/User_Guide/Settings/dedicated_ip_warmup.html) is the gradual process of establishing a reputation as a legitimate email sender in the eyes of ISPs (Internet Service Providers). When an ISP observes email suddenly coming from a new or “cold” (recently dormant) IP address, they will take notice and immediately begin evaluating the traffic coming from that IP.
+IP warming is the gradual process of establishing a reputation as a legitimate email sender in the eyes of ISPs (Internet Service Providers). When an ISP observes email suddenly coming from a new or “cold” (recently dormant) IP address, they will take notice and immediately begin evaluating the traffic coming from that IP.
```

Comment:
Just out of curiosity, should it be:

ISPs (Internet Service Providers)

OR
Internet Service Providers (ISPs)?

Just want to make sure I'm not doing it wrong!